cfgov/unprocessed/js/apps/financial-well-being/fwb-questions.js

```diff
@@ -0,0 +1,83 @@
+/* eslint no-global-assign: "off" */
+'use strict';
+
+var Analytics = require( '../../modules/Analytics' );
+
+function init() {
+
+  var totalQuestions = 12;
+  var quizComplete = false;
+  var questionValues = {};
+  var radioButtons = document.querySelectorAll( '[type="radio"]' );
+  var submitButton = document.querySelector( '#submit-quiz' );
+
+  function allQuestionsCompleted() {
+    return Object.keys( questionValues ).length === totalQuestions;
+  }
+
+  function enableSubmit() {
+    submitButton.title = 'Get your score';
+
+    submitButton.classList.remove( 'a-btn__disabled' );
+  }
+
+  function handleRadio( input ) {
+    if ( input.name && input.checked ) {
+
+      questionValues[input.name] = input.value;
+    }
+
+    quizComplete = allQuestionsCompleted();
+
+    if ( quizComplete ) {
+      enableSubmit();
+    }
+  }
+
+  function sendEvent( action, label, category ) {
+    var eventData = Analytics.getDataLayerOptions( action, label, category );
+    Analytics.sendEvent( eventData );
+  }
+
+  submitButton.addEventListener( 'click', function( event ) {
+    // Confirm that all questions are answered
+    if ( !quizComplete ) {
+      // If not, block the form submission action
+      return event.preventDefault();
+    }
+
+    // Otherise, allow the form submission to go through as normal
+    return null;
```

Comment:
Does this need to return `null` for the form to submit? Otherwise the returns could be stripped out here and on L:36. Ideally the function returns the same type, either undefined (when no returns are set) or a boolean. Right now it returns `undefined` or `null`, which isn't so helpful on what it has done.

Reply:
Removed in the next PR.